polyglot/upload.py

```diff
@@ -122,7 +124,7 @@ def filter_utxos_for_bcat(self):
         """filters out all utxos with 0 conf or too low amount for use in a BCAT part transaction"""
         filtered_utxos = []
         for utxo in self.get_unspents():
-            if utxo.confirmations != 0 and utxo.amount >= 100000:
+            if utxo.confirmations >= self.utxo_min_confirmations and utxo.amount >= self.fee * 100000:
```

Comment:
I think 100000 should be changed to a constant like `MAX_DATA_CARRIER_SIZE` or something... to make it clearer what is going on here.

Do you think that would be better?